.github/workflows/botonic-nlu-tests.yml

```diff
@@ -1,11 +1,17 @@
 name: Botonic nlu tests
  
-on: [push]
+on:
+  push:
+    paths:
+      - 'packages/botonic-nlu/**'
+      - '.github/workflows/botonic-nlu-tests.yml'
  
 jobs:
   botonic-plugin-nlu-tests:
     name: Botonic nlu tests
     runs-on: ubuntu-latest
+    env:
+      PACKAGE: botonic-nlu
     steps:
     - name: Checking out to current branch (Step 1 of 7)
       uses: actions/checkout@v1
```

Comment:
make the most of this PR to try to upgrade to actions/checkout@v2, actions/cache@v2 and actions/setup-node@v2
luckily you can do a replace-all for this

Reply:
updated in https://github.com/hubtype/botonic/pull/1177/commits/6076e6a62da605e85b0a648245c0014480f2ac7e